syntax = "proto3";

package jim.ds.storage.pb;

message InternalTag {
    string txn_id = 1;
}
